docs/pages/includes/install-linux.mdx

```diff
@@ -157,41 +198,74 @@ Use the appropriate commands for your environment to install your package.
-  Selecting previously unselected package teleport-ent.
-  (Reading database ... 30810 files and directories currently installed.)
-  Preparing to unpack teleport-ent_(=cloud.version=)_$SYSTEM_ARCH.deb ...
-  Unpacking teleport-ent (=cloud.version=) ...
-  Setting up teleport-ent (=cloud.version=) ...
+  # Download Teleport's PGP public key
+  $ sudo curl https://apt.releases.teleport.dev/gpg \
+  -o /usr/share/keyrings/teleport-archive-keyring.asc
+  # Source variables about OS version
+  $ source /etc/os-release
+  # Add the Teleport APT repository for v(=teleport.major_version=). You'll need to update this
+  # file for each major release of Teleport.
+  # Note: if using a fork of Debian or Ubuntu you may need to use '$ID_LIKE'
+  # and the codename your distro was forked from instead of '$ID' and '$VERSION_CODENAME'.
+  # Supported versions are listed here: https://github.com/gravitational/teleport/blob/master/build.assets/tooling/cmd/build-os-package-repos/runners.go#L42-L67
+  $ echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] \
+  https://apt.releases.teleport.dev/${ID?} ${VERSION_CODENAME?} stable/cloud" \
+  | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
+
+  $ sudo apt-get update
+  $ sudo apt-get install teleport-ent
   ```
 
   </TabItem>
-  <TabItem label="Amazon Linux 2/RHEL (RPM)" scope="cloud">
+  <TabItem label="AL 2/RHEL 7- (yum)" scope="cloud">
+
+  ```code
+  # Source variables about OS version
+  $ source /etc/os-release
+  # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
+  # file for each major release of Teleport.
+  # Note: if using a fork of RHEL/CentOS or Amazon Linux you may need to use '$ID_LIKE'
+  # and the codename your distro was forked from instead of '$ID'
+  # Supported versions are listed here: https://github.com/gravitational/teleport/blob/master/build.assets/tooling/cmd/build-os-package-repos/runners.go#L133-L153
+  $ sudo yum-config-manager --add-repo "$(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/cloud/teleport.repo")"
+  $ sudo yum install teleport-ent
+  #
+  # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
+  # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
+  ```
 
-  After Downloading the `.rpm` file for your system architecture, install it with `rpm`:
+  </TabItem>
+  <TabItem label="AL 2023/RHEL 8+ (dnf)" scope="oss">
 
   ```code
-  $ rpm -i ~/Downloads/teleport-ent-(=cloud.version=).<Var name="$SYSTEM-ARCH"/>.rpm
-  warning: teleport-ent-(=cloud.version=).$SYSTEM-ARCH.rpm: Header V4 RSA/SHA512 Signature, key ID 6282c411: NOKEY
+  # Source variables about OS version
+  $ source /etc/os-release
+  # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
+  # file for each major release of Teleport.
```

Comment:
As above, `stable/cloud` is automatically updated across major versions so we shouldn't reference `teleport.major_version` here or advise that people need to update it